### DISCUSSION TITLE
Which PIPED servers are LibreTube compatible with?

### DISCUSSION THREAD

**Discussion by nanedi** [libre-tube/LibreTube#569] [General]
Hi

Which YouTube PIPE servers are LibreTube compatible with?

I want to install my own YouTube PIPE server.

Thanks

**Comment by FireMasterK** [comment]
These: https://github.com/TeamPiped/Piped/wiki/Instances

**Comment by Bnyro** [comment]
Any in theory since you can now add custom instances. Self-hosting isn't a bad thing :)

**Reply by nanedi** [reply]
> Any in theory since you can now add custom instances. Self-hosting isn't a bad thing :)

Yea~ I can get it to have a faster internet connection.
